### ISSUE TITLE
OnexPlayer AMD 5700u unit - black screen 

### ISSUE THREAD

**Issue by Smee342** [holoiso-eol/holoiso#308]
I managed to install Steam)S 3.0. but once I reboot after installing all I get a black. I did find someone who said to do the tab space enter keys. that did not work. any help would be appreciated. 

Thanks.

**Comment by ruineka** [comment]
Try hooking up an external display, I'm finding that a lot of issues with holoiso are resolved by hooking up a display a few times and disconnecting/rebooting etc.

**Comment by Smee342** [comment]
> Try hooking up an external display, I'm finding that a lot of issues with holoiso are resolved by hooking up a display a few times and disconnecting/rebooting etc.

I bought a usbc to hdmi. When I plug it into my tv I am still just getting a double black screen. I even tried to install steam os with it plugged in and still same results. Any other tips? 

Thanks

**Comment by konradknitter** [comment]
Did you ever get Setup DeckUI?

I had an issue when I setup blank user/password during install that it didn’t boot. 
After setting root/username password to something it got booted into initial setup.

On another I heard people got issues after initial setup had problems - but myself I didn’t experienced it.

**Comment by Smee342** [comment]
I> Did you ever get Setup DeckUI?
> 
> 
> 
> I had an issue when I setup blank user/password during install that it didn’t boot. 
> 
> After setting root/username password to something it got booted into initial setup.
> 
> 
> 
> On another I heard people got issues after initial setup had problems - but myself I didn’t experienced it.


I did, found a thread on Reddit about using just lowercase letter for username and password in setup. This got me passed the black screen.

Then when steam is loaded I got no WiFi. Found another thread that said to load Holoiso instead of the Neptune. So that got me in to steam Os with WiFi.

Next obstacle was my screen was upside down. Someone on Reddit directed me to gamescope - onexplayer. So I ram that and got my screen, and everything else working. 

As of this morning my onexplayer is playing games via Steam Os.
